src/components/Book.js

```diff
@@ -1,29 +1,85 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import { removeBook, deleteBook } from '../redux/features/books/booksSlice';
 
+import 'react-circular-progressbar/dist/styles.css';
+
 function Book({
   itemId, title, author, category,
 }) {
+  const [chapterNumber, setChapterNumber] = useState({
+    chapter: 0,
+    percent: 0,
+  });
   const dispatch = useDispatch();
 
   const remover = (itemId) => {
     dispatch(deleteBook(itemId)).then(() => dispatch(removeBook(itemId)));
   };
   return (
-    <div>
-      <p>{category}</p>
-      <h2>{title}</h2>
-      <p>{author}</p>
+    <div className="book-container">
+      <div className="book-item-main">
+        <p className="category">{category}</p>
+        <h2>{title}</h2>
+        <p className="author">{author}</p>
+        <button type="submit" className="button-show">
+          Comments
+        </button>
+        <button
+          type="submit"
+          className="button-show remove"
+          onClick={() => remover(itemId)}
+          z
+        >
+          Remove
+        </button>
+        <button type="submit" className="button-show">
+          Edit
+        </button>
+      </div>
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <span>
+          {chapterNumber.percent < 100 ? (
+            <CircularProgressbar
+              className="circular-progressbar"
+              value={chapterNumber.percent}
+              text={`${chapterNumber.percent} %`}
+            />
+          ) : (
+            <CircularProgressbar
+              className="circular-progressbar"
+              value={100}
+              text={`${100} %`}
+            />
+          )}
+        </span>
+        <span style={{ marginLeft: '1rem' }}>Completed</span>
+      </div>
+
+      <div className="book-item">
+        <h4>Current Chapter</h4>
+        {chapterNumber.chapter < 10 ? (
+          <h2>{`Chapter ${chapterNumber.chapter}`}</h2>
+        ) : (
+          <h2>Completed</h2>
+        )}
 
-      <button type="submit" onClick={() => remover(itemId)}>
-        Delete
-      </button>
+        <button
+          type="submit"
+          className="update"
+          onClick={() => setChapterNumber({
+            chapter: chapterNumber.chapter + 1,
+            percent: chapterNumber.percent + 10,
+          })}
+        >
+          Update Progress
+        </button>
+      </div>
```

Comment:
- [ ] As I mentioned above :point_up_2: please follow [Zaplin](https://zeplin.io/) design but your font styles and colours are not the same as [Zaplin](https://zeplin.io/) design so kindly fix this issue as well :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-05-13 16-01-16](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/aa6b233a-4be3-4cff-99ea-4f7632dd27f2) | ![Screenshot from 2023-05-13 16-01-27](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/cb32c30e-f7e7-4937-aeae-515d2f55c76c) |

Reply:
@aamirkhan2478  I understand everything but I had problems with the Zeplin login . Thank you for your feedback